tests/test_ancestry.py

```diff
@@ -178,6 +178,46 @@ def test_multimerger(self):
         )
         self.verify(sim, multiple_mergers=True)
 
+    def test_gene_conversion_n25(self):
```

Comment:
We're not actually checking for  properties of the GC ARG here. We can either update the ``verify`` method above to also check for GC nodes, or make a separate test class where the verify is just checking for properties of the ARG with GC. I don't mind which.

Reply:
Well spotted. I've extended the `verify` method to check GC nodes. The alternative would have resulted in a lot of duplicated code.